test/e2e/cluster/cluster.go

```diff
@@ -76,7 +77,13 @@ func (h *hybridCluster) create(ctx context.Context, client *eks.Client, logger l
 	logger.Info("Successfully started EKS hybrid cluster")
 	logger.Info(awsutil.Prettify(cluster))
 
-	return nil
+	podIdentityAddon := addon.Addon{
+		Cluster:       h.Name,
+		Name:          "eks-pod-identity-agent",
+		Configuration: "{\"daemonsets\":{\"hybrid\":{\"create\": true}}}",
+	}
+
+	return podIdentityAddon.Create(ctx, client, logger)
```

Comment:
IMO this makes more sense in `cluster/create.go`, that's the part that groups the cluster creation itself with the creation of additional supporting constructs that together form the hybrid cluster.